### DISCUSSION TITLE
Enemies for Tropical Paradise

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2910] [Ideas]
these ideas aren't for v0.7.0, they'll be for v0.8.0 where Tropical Paradise will be released.

1. Tropical Fish
Tropical fish will appear on the beaches. For the beach style, there will be fish such as orange with white stripes (basically nemo), light blue fish, the chasing fish of Icy Island with another tropical variant, which would be the lionfish, a puffer fish that when you get close, explodes small spikes. a mix of Dive mine and Big Granite Corrupted.

For those in the jungle, it will be lionfish, an angler fish that was seen in an official concept of the supertux discord, the dropfish (i.e. the ugliest fish in the world) that could be asleep, but when it sees tux, it wakes up and chases after it, something similar to mario world

2. More Crab Variants

The first crab was invented by me, the big crab claws. A crab that has large claws that could kill tux with a pinch would also protect fireballs with its claws. The only thing that can kill him is either buttjump him or be killed by a crusher. These would appear in the deepest jungles. The crab will be a brown color

The second crab is a blue crab, the name is sea crab. By name he will appear in waters, and his attacks are to jump and pinch you.

The third and last one is yellow, they are called mini crabs. They don't do anything, they just walk and jump in places, but they're smart not to fall off a cliff

3. Annoying Monkeys

These spawn in jungles and on beaches, will throw coconuts and enemies. Basically a new variant of the dropper. Monkeys will be able to throw, coconuts, crabs, snails, and objects

These ideas are only three, as I don't want to extend this any further, but in the future I will be able to give more ideas.

**Comment by veebhavsupertuxfan19o3** [comment]
These are interesting and i LOOVE the idea of monkeys although i think by now its not really needed due to the new Supertux tales addon.Althogh a crab sounds interesting. But personally i dont really think we need a tropical fish cuz like...we already have two jumping fish variants.And some very VERY annoying swimming fish like the piranhas that chase after you.
